dcos/config.py

```diff
@@ -203,9 +203,15 @@ def get_config_schema(command):
             pkg_resources.resource_string(
                 'dcos',
                 'data/config-schema/core.json').decode('utf-8'))
-
-    executable = subcommand.command_executables(command)
-    return subcommand.config_schema(executable)
+    elif command in subcommand.default_subcommands():
+        resource = "data/config-schema/{}.json".format(command)
+        return json.loads(
+            pkg_resources.resource_string(
+                'dcoscli', resource).decode('utf-8'))
+        return subcommand.default_config_schema(command)
```

Comment:
There are two return statements. I like the second one. We don't need `resource` if we do the second one.